### ISSUE TITLE
Unclear usage and status  of work_type and work_genre for display in the frontend

### ISSUE THREAD

**Issue by GoogleCodeExporter** [vsimko/gama-gateway#11]
```
In the rdf schema the  work_genre  property is described as :  

"   GAMA Metadata Schema provides two levels of genre definition based on
WorkType. All GAMA content partners use their own system for categorising
works in their collections. Some partners use genres, others use types and
some content partners even have a subdivision of subtypes to specify the
nature of works they are describing. These co-existing solutions for
structures and vocabularies serve to describe the individual collections as
precisely as possible; a similar and compatible solution has to be found
for GAMA as well. "

There is also work_type, which resembles 'genre' but seems to be used for a
technical classification eg 'video' 

In the current devel database there are no works with  work_types and
genres defined. For further frontend  UI development the followoing 
questions have to be answered: 

- what values to expect for work_type ? 
- will every work have a work_type defined ? 
- What values to expect for Genre ? Will every Work / Manif have a genre ? 
- hwo are the normalized GAMa genres stored ? How to distinguish them from
the CP-spefici genres ?    

What actions are needed to get genres/types in the rdf repository?  is it
-- db adaper development  ?
-- RDf schema extension or refinement  ?
-- CP database  completion of empty fields  ?
-- new import actions of existing data. ?
-- harmonization tool development?
-- harmonization actions?
--other, .... ?

In short, what actiosn are needed to eb able to display genres (
nortmalized or CP-specific) in the fronend ?

Note that genres are very valuable to sort, classify, and filter search
resuls in the frontend.    

```

Original issue reported on code.google.com by `toon...@gmail.com` on 5 Mar 2009 at 2:35
